app/services/migration/migrators/application.rb

```diff
@@ -124,6 +129,10 @@ def touch_updated_at?(application, ecf_npq_application)
       false
     end
 
+    def ecf_profile_school_urn_or_ecf_npq_application_school_urn?(ecf_npq_application)
+      ecf_npq_application.profile&.school_urn.presence || ecf_npq_application.school_urn
```

Comment:
when including the profile school urn, some schools don't exist with some urns (around 6). Can we add a method which finds the school id but doesn't fail if it can't find it, we can also keep the existing method as its needed elsewhere for finding schools and failing